### PR TITLE
squid:S1155 - Collection.isEmpty() should be used to test for emptiness

### PR DESCRIPTION
**Pull request by georgekankava**
This pull request is focused on resolving occurrences of Sonar rule
squid:S1155 - Collection.isEmpty() should be used to test for emptiness.
You can find more information about the issue here:
https://dev.eclipse.org/sonar/rules/show/squid:S1155
Please let me know if you have any questions.
George Kankava

### CODE REVIEW

**Comment by mswiderski** [comment]
ok to test

**Comment by mswiderski** [comment]
jenkins retest this

File: jbpm-simulation/src/main/java/org/jbpm/simulation/converter/SimulationFilterPathFormatConverter.java [hunk 1]
```diff
@@ -92,7 +92,7 @@ public SimulationFilterPathFormatConverter(SimulationDataProvider provider) {
                   simPath.addBoundaryEventId(fe.getId());  
                 } else if (fe instanceof CatchEvent) {
                     CatchEvent act = (CatchEvent) fe;
-                    if(act.getIncoming() == null || act.getIncoming().size() == 0 && !isParentEventSubprocess(fe)) {
+                    if(act.getIncoming() == null || !act.getIncoming().isEmpty() && !isParentEventSubprocess(fe)) {
```
Comment (jakubschwan) [inline jbpm-simulation/src/main/java/org/jbpm/simulation/converter/SimulationFilterPathFormatConverter.java]: There should not be "**!**" before `isEmpty()`.

**Comment by georgekankava** [comment]
@mswiderski Sorry about that. PR Updated.

**Pull request merged**